### PR TITLE
Update contribution guide

### LINKED ISSUE #24: Document test process for operator
Document how to test operator using molecule and how to debug the tests.

## Acceptance criteria

* Add devel environment setup to documentation
* Add testing guide to documentation
* Add debug info to documentation
* Add notes about possible debug issues (docker volume size)

## Definition of Done

* Acceptance criteria are met
* New chapter in documentation - developers guide

### PR DESCRIPTION
**Pull request by Zlopez**
Closes #24 

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -28,6 +28,37 @@ Tests can be run using either molecule or operator-sdk cli (which uses molecule
...
+   molecule converge -s test-local #runs local test without destroy sequence
+   docker ps #find container named kind-test-local
+   docker exec -it <container_id> bash #<container_id> of container from previous command
+   kubectl config set-context --current --namespace=osdk-test #sets namespace to operator-sdk
+
+Here are few useful commands for debugging, for another commands look at `kubectl help`:
+
+.. code-block:: bash
+
+   kubectl get all #returns all resources in the current namespace
```
Comment (odra) [inline docs/source/contributing.rst:47]: I think it is worth adding a note that it won't list ingresses here

**Comment by odra** [comment]
just one comment, looks good.

File: docs/source/contributing.rst (hunk 1 above)
Comment (Zlopez) [reply docs/source/contributing.rst:47]: I forgot about this, I will add command for listing the ingress. Was it `kubectl get routes`?

Comment (odra) [reply docs/source/contributing.rst:47]: `k get ingress` - route is only available in openshift

**Pull request merged**